articles/machine-learning/how-to-prepare-datasets-for-automl-images.md

```diff
@@ -50,6 +50,10 @@ If you already have a data labeling project and you want to use that data, you c
 ### Using pre-labeled training data
 If you have previously labeled data that you would like to use to train your model, you will first need to upload the images to the default Azure Blob Storage of your Azure ML Workspace and register it as a data asset. 
 
+Below script uploads the image data at path "./data/odFridgeObjects" on your local machine to Azure Blob Storage. Thereafter, it creates a new data asset with the name "fridge-items-images-object-detection" in your Azure ML Workspace. Datastore property of this newly created data asset is made to point to location in Azure Blob Storage where the image data was uploaded. 
```

Comment:
minor:  Below script uploads the image data at path "./data/odFridgeObjects" on your local machine to Azure Blob Storage. --> The below script uploads the image data on your local machine at path "./data/odFridgeObjects" to Azure Blob Storage.